barretenberg_wrapper/build.rs

```diff
@@ -219,12 +220,11 @@ fn link_lib_omp() {
     //
     // we are using clang, so we need to tell the linker where
     // to search for lomp.
-    //
-    // For macOS, we do not need to supply the search path
-    // however for linux we do
     if let OS::Linux = select_os() {
         let llvm_dir = find_llvm_linux_path();
         println!("cargo:rustc-link-search={}/lib", llvm_dir);
+    } else if let ARM_APPLE = select_toolchain() {
```

Comment:
Since we're just checking string equality we don't need to use pattern matching here. Should be `else if ARM_APPLE == select_toolchain()` or the reverse `select_toolchain() == ARM_APPLE`

Reply:
Good point. Rewrote matching in the new commit including Homebrew path for Intel Macs.